roles/openshift_metering/tasks/operator-install.yml

```diff
@@ -193,3 +193,7 @@
     name: "{{ mktemp.stdout }}"
     state: absent
   changed_when: False
+
+- name: Install monitoring resources
+  import_tasks: monitoring-install.yml
+  when: openshift_metering_enable_monitoring
```

Comment:
Sorry I didn't catch this earlier, but we always need to cast to bool for user-supplied boolean values.
Often times, they'll supply 'false' instead of False, and that will eval to true by default.

openshift_metering_enable_monitoring | bool

Reply:
Oh totally I knew this, whoops.

Reply:
Fixed up